tests/www/test_views.py

```diff
@@ -319,6 +319,11 @@ def test_not_found(self):
         resp = self.client.get('/', follow_redirects=True)
         self.assertEqual(resp.status_code, 404)
 
+    def test_robots_txt(self):
+        resp = self.client.get('/test/robots.txt', follow_redirects=True)
```

Comment:
Does /test/robots.txt do anything? I thought this was  only read if it lives at /robots.txt?

Reply:
@ashb we set the default url in the config for the test to include `/test/`
https://github.com/apache/airflow/blob/master/tests/www/test_views.py#L303

Reply:
But serving /test/robots.txt won't do anything - nothing will ever request it.